domains/Sylestic.json

```
@@ -0,0 +1,9 @@
+{
+   "owner":{
+      "username":"Sylestic",
+      "email":"sylestic@protonmail.com"
+   },
+   "record":{
+      "CNAME":"sylestic.github.io"
```

Maybe you want to redirect it to https://sylestic.ml/?
```suggestion
      "URL":"sylestic.ml"
```

---

Thanks!